src/main/scala/net/jcazevedo/moultingyaml/YamlPrinter.scala

```diff
@@ -81,15 +81,20 @@ abstract class YamlPrinter(
   def apply(value: YamlValue): String
 }
 
-class SnakeYamlPrinter(flowStyle: FlowStyle,
-                       scalarStyle: ScalarStyle,
-                       lineBreak: LineBreak)
+class SnakeYamlPrinter(flowStyle: FlowStyle = FlowStyle.DEFAULT,
+                       scalarStyle: ScalarStyle = ScalarStyle.DEFAULT,
+                       lineBreak: LineBreak = LineBreak.DEFAULT)
     extends YamlPrinter(flowStyle, scalarStyle, lineBreak) {
-  override def apply(value: YamlValue): String = {
+
+  def dumperOptions : DumperOptions = {
```

Comment:
Can you remove the extra space between `dumperOptions` and the colon? Just hitting `compile` should be enough for Scalariform rules to kick in.